build/Build.DetermineAffectedTests.cs

```diff
@@ -13,15 +13,19 @@ namespace Calamari.Build;
 partial class Build
 {
     [Parameter(Name = "DefaultGitBranch")] readonly string MainBranchName;
-    
+
     [PublicAPI]
     Target DetermineAffectedTests =>
         target => target
             .Executes(() =>
             {
                 if (GitVersionInfo is null)
                     throw new ArgumentNullException(nameof(GitVersionInfo));
-                
+
+                File.WriteAllText("placeholder.txt",
+                    "TeamCity can't stand to have no artifacts in a consuming configuration, so this is a placeholder");
```

Comment:
I can feel the pain in this 😅

Reply:
:joy: The words are taken from someone elses previous work, but the feeling is all mine now